eval.go

```diff
@@ -2179,6 +2207,22 @@ func (e *callExpr) eval(app *app, args []string) {
 				app.ui.loadFile(app, true)
 				app.ui.loadFileInfo(app.nav)
 			}
+		case "mkdir: ":
+			app.ui.cmdPrefix = ""
+			filepath := filepath.Clean(replaceTilde(s))
+			if err := os.MkdirAll(filepath, os.ModePerm); err != nil {
+				app.ui.echoerrf("mkdir: %s", err)
+				return
+			}
+			if gSingleMode {
+				app.nav.renew()
+				app.ui.loadFile(app, true)
+			} else {
+				if err := remote("send load"); err != nil {
+					app.ui.echoerrf("rename: %s", err)
```

Comment:
I went through the code one last time, just in case, I think this is a copy/paste error:

```suggestion
					app.ui.echoerrf("mkdir: %s", err)
```

Reply:
That's right, fixed.